themes/jb/layouts/partials/header.html

```diff
@@ -96,8 +96,13 @@
                       </div>
                       {{ else if eq .Identifier "home"}}
                       <!-- do nothing, home is done as "navbar-brand" class above outside of "navbar-menu" -->
+                      {{ else if eq .Identifier "live"}}
+                          <a id="livebutton" href="{{ .URL }}" {{ if (hasPrefix .Identifier "ext-") }}target="_blank"{{ end }} class="navbar-item {{ if $active }}active{{ end }}">
+                              {{ .Pre }}
+                              <span>{{ .Name }}</span>
+                          </a>
                       {{ else }}
-                          <a href="{{ .URL }}" {{ if (hasPrefix .Identifier "ext-") }}target="_blank"{{ end }} class="navbar-item {{ if $active }}active{{ end }}">
+                          <a href="{{ .URL }}" class="navbar-item {{ if $active }}active{{ end }}">
```

Comment:
So, it looks like you actually removed the wrong if section. You removed the one that was in the `else` block instead of your `else if` block that I'd made the suggestion for. :upside_down_face:. Just wanted to make sure we didn't break this functionality for the other links :sweat_smile: 

Also, in the future there's actually a way in the GitHub UI allowing you to accept it [from the GitHub web-UI](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes). Whenever we do the suggestions you don't even have to leave the web-ui (unless you want to test, which is a good idea after accepting them :upside_down_face: ), but just wanted to mention it :grin: 



```suggestion
                      {{ else if eq .Identifier "live"}}
                          <a id="livebutton" href="{{ .URL }}" class="navbar-item {{ if $active }}active{{ end }}">
                              {{ .Pre }}
                              <span>{{ .Name }}</span>
                          </a>
                      {{ else }}
                          <a href="{{ .URL }}" {{ if (hasPrefix .Identifier "ext-") }}target="_blank"{{ end }} class="navbar-item {{ if $active }}active{{ end }}">
```